scripts/components/ZoomButtons/ZoomButtons.js

```diff
@@ -9,6 +10,162 @@ export default class ZoomButtons extends React.Component {
+    }
+
+    if (!this.props.isZoomOutDisabled) {
+      this.buttons['zoomOut'] = this.zoomOutButtonRef;
+    }
+  }
+
+  /**
+   * Get index of button.
+   * @param {string} type Button type.
+   * @returns {number} Button index.
+   */
+  getIndex(type) {
+    if (typeof type !== 'string' || Object.keys(this.buttons).length == 0) {
+      return;
+    }
+    return Object.values(this.buttons).indexOf(this.buttons[type]);
+  }
+
+  /**
+   * Determine if button should get focus.
+   * @param {string} type Button type.
+   * @returns {boolean} True, if button should get focus.
+   */
+  getButtonFocus(type) {
+    if (typeof type !== 'string') {
+      return false;
+    }
+
+    return this.getIndex(type) === this.state.focusButton;
+  }
+
+  /**
+   * Get tabindex for button.
+   * @param {string} type Button type.
+   * @returns {number} Tabindex.
+   */
+  getTabIndex(type) {
+    if (typeof type !== 'string') {
+      return -1;
+    }
+
+    return this.getIndex(type) === this.state.currentButtonIndex && !this.props.isHiddenBehindOverlay ? 0 : -1;
+  }
+
+  /**
+   * Component did update.
+   * @param {object} prevState Previous state.
+   */
+  componentDidUpdate(prevState) {
+    this.addButtonRefs();
+
+    if (this.props.isZoomInDisabled && this.state.currentButtonIndex === this.getIndex('zoomIn')) {
+      this.moveFocus(1);
+    }
+
+    if (this.props.isZoomOutDisabled && this.state.currentButtonIndex === this.getIndex('zoomOut')) {
+      this.moveFocus(-1);
+    }
+
+    if (this.state.focusButton !== null && this.state.focusButton !== prevState.focusButton) {
+      if (this.zoomInButtonRef.current || this.zoomOutButtonRef.current) {
+        this.handleFocus();
+      }
+    }
+  }
+
+  /**
+   * Move button focus.
+   * @param {number} offset Offset to move position by.
+   */
+  moveFocus(offset) {
+    if (typeof offset !== 'number') {
+      return;
+    }
+
+    const newIndex = this.state.currentButtonIndex + offset;
+  
+    if (newIndex < 0 || newIndex >= Object.keys(this.buttons).length) {
+      return;
+    }
+
+    if (this.props.isZoomInDisabled && newIndex === this.getIndex('zoomIn')) {
+      return;
+    }
+
+    if (this.props.isZoomOutDisabled && newIndex === this.getIndex('zoomOut')) {
+      return;
+    }
+
+    this.setState({
+      currentButtonIndex: newIndex,
+      focusButton: newIndex
+    });
+  }
+
+
+  /**
+   * Handle focus on zoom buttons.
+   */
+  handleFocus() {
+    Object.values(this.buttons)[this.state.currentButtonIndex].current.focus();
+  }
+
+  /**
+   * Handle blur on zoom buttons.
+   */
+  handleBlur() {
+    this.setState({ focusButton: null });
+  }
+
+  /**
+   * Handle key down event.
+   */
+  handleKeyDown(event) {
+    switch (event.key) {
+      case 'ArrowDown':
+      case 'ArrowRight':
+        this.moveFocus(1);
+        break;
+      case 'ArrowUp':
+      case 'ArrowLeft':
+        this.moveFocus(-1);
+        break;
+      case 'Home':
```

Comment:
This sets the tabindex correctly, but the respective button does not get focus.

Reply:
Should be fixed now